gdk-sys/Cargo.toml

```diff
@@ -1,5 +1,6 @@
 [build-dependencies]
 pkg-config = "0.3.7"
```

Comment:
This will cause a CI failure: https://github.com/gtk-rs/gir/pull/937/files#r446032219

@EPashkin What do you suggest?

Reply:
Apart from that everything works

Reply:
https://github.com/gtk-rs/gir/pull/941

Reply:
Ah, @EPashkin fixed already :)